cli/command/deploy_command.go

```diff
@@ -115,8 +115,17 @@ func (d *DeployCommand) printDeploys(deploys []*models.Deploy) error {
 	return d.Printer.PrintEntities(entities)
 }
 
-func filterDeploys(deploys []*models.Deploy) []*models.Deploy {
-	catalog := map[string]*models.Deploy{}
+func (d *DeployCommand) printDeploySummaries(deploys []*models.DeploySummary) error {
+	entities := []entity.Entity{}
+	for _, s := range deploys {
+		entities = append(entities, entity.NewDeploySummary(s))
+	}
+
+	return d.Printer.PrintEntities(entities)
```

Comment:
I think so. We don't have any consumers of the API (cli, terraform) that display things in pages.